forch/faucetizer.py

```diff
@@ -91,21 +91,23 @@ def process_devices_state(faucetizer: Faucetizer, devices_state: DevicesState):
 
 def load_faucet_config(file):
     """Load network state file"""
-    LOGGER.info('Loading faucet config file %s', file)
     with open(file) as config_file:
         faucet_config = yaml.safe_load(config_file)
-    LOGGER.info('Loaded faucet config from %s', file)
     return faucet_config
```

Comment:
nb: you can inline the return if you want -- no need for the local variable in this case